stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/messages/ThreadParticipants.kt

```diff
@@ -62,12 +62,14 @@ public fun ThreadParticipants(
                 index + 1
             }.toFloat()
 
-            ImageAvatar(
+            Avatar(
                 modifier = itemPadding
                     .zIndex(itemPosition)
                     .size(itemSize)
                     .border(border = borderStroke, shape = ChatTheme.shapes.avatar),
-                painter = painter
+                imageUrl = user.image,
+                initials = user.initials,
+                textStyle = ChatTheme.typography.captionBold.copy(fontSize = 7.sp),
```

Comment:
I don't want to introduce a new font just for the one use case. Seems like an exceptional use case and in general a font shouldn't be that small.

Reply:
Agreed - this should be fine for now.